roles/reproducer/tasks/main.yml

```diff
@@ -128,9 +122,15 @@
         cmd: >-
           ansible-playbook -i ~/ci-framework-data/artifacts/zuul_inventory.yml
           -e @~/reproducer-variables.yml
-          cifmw-bootstrap.yml
+          -e @scenarios/reproducers/networking-definition.yml
+          playbooks/01-bootstrap.yml
 
     - name: Emulate CI job
       when:
         - cifmw_job_uri is defined
       ansible.builtin.import_tasks: ci_job.yml
+
+    - name: Prepare VA deployment
+      when:
+        - cifmw_architecture_va_scenario is defined
```

Comment:
For me this one seems confusing, cause when a user calls the reproducer to fire up a new environment, he/she needs to pass something like this at the command line `-e cifmw_architecture_va_scenario=hci` (or put it in an env file). That, explicitly tells the user that the deployment will be for that particular VA, what is completely wrong, as the env can be used for whatever VA/DT they want (if it's layout matches the requirements, of course).

Reply:
It's not wrong - this is automation, for the records. If they don't want to get any VA in place, so be it, just don't pass anything and build the needed bits on their own.
That parameter should be passed via the scenarios/reproducers/VA_NAME.yml environment file, in order to allow a full automation.
Don't forget that we don't have only VA. We therefore have to get something to switch the path between "non-VA/DT" vs "VA/DT".

That said, as done in the previous patch, removing the `_va` from the param name may be a good move, since we may support things that aren't VA, if properly described in the architecture repository.

Reply:
(TODO: rename top-level parameter)